app/services/usps_in_person_proofing/eipp_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module UspsInPersonProofing
+  class EippHelper
```

Comment:
@gina-yamada @KeithNava To ensure that we're consistent in how we determine if a user is going through EIPP, I thought it would be helpful to have a central file, like this `EippHelper`.  Let me know if you think there's a more efficient way to centralize this logic, or if you have suggested refactors for the contained methods.